dev.Cogitri.Health.json

```diff
@@ -34,38 +34,52 @@
         "append-path": "/usr/lib/sdk/rust-stable/bin",
         "env": {
             "CARGO_HOME": "/run/build/Health/cargo",
-            "RUSTFLAGS" : "-L=/app/lib",
+            "RUSTFLAGS": "-L=/app/lib",
             "RUST_BACKTRACE": "1"
         }
     },
-    "modules" : [
+    "modules": [
         {
```

Comment:
You could probably drop libadwaita/libsass/sassc as they are included in GNOME SDK 42.

Reply:
Thanks for the hint! Just pushed it without those libraries. Let's see if it builds.

Reply:
Make sure it runs as well, just in case ;)

Reply:
nice, works :smile: